rust/lancedb/src/query.rs

```
@@ -831,6 +837,14 @@ impl VectorQuery {
         self
     }
 
+    /// Set the distance range for vector search,
+    /// only rows with distances in the range [lower_bound, upper_bound) will be returned
+    pub fn distance_range(mut self, lower_bound: Option<f32>, upper_bound: Option<f32>) -> Self {
+        self.lower_bound = lower_bound;
+        self.upper_bound = upper_bound;
```

Should we return an error if both `lower_bound` and `upper_bound` are `None`?

---

I'd prefer don't return an error for this case
It's equal to be without any distance threshold, or "revert the distance range parameter"